agreements/project-contribution/terms-and-conditions.md

```diff
@@ -166,6 +166,8 @@ Contributions or donations held in support of these purposes must be transferred
 These identifiers must meet the Sponsor’s quality and trademark standards and be approved before use.
 Once approved, these identifiers will be added to the Sponsor’s [Trademark List][trademarks], with all goodwill benefiting the Sponsor.
 
+**Trademark-related costs**, including registration, renewal, and protection, are project expenses. These costs are the responsibility of the Project and are typically paid from the Project’s restricted fund or by sponsors. All trademark-related legal work must be coordinated through CF’s designated contacts. Details are provided in the [Trademark Management and Costs Policy](../policies/trademark-management-costs.md).
+
```

Comment:
Is this a change or a clarification?

Reply:
I've updated the language in trademark costs: 

> Projects are expected to help ensure there are sufficient funds to maintain any registered trademarks associated with their work.  
Trademark-related costs are normally paid from the Project’s restricted fund under its [Fiscal Sponsorship Agreement](../fiscal-sponsorship/terms-and-conditions.md).  
Sponsors may also choose to cover these costs on behalf of a Project.

This more closely matches my intent..